app/src/main/java/com/kms/cura/view/activity/DoctorViewActivity.java

```diff
@@ -349,23 +354,53 @@ private void hideProgressDialog() {
 
     @Override
     public void handleEvent(String event, Object data) {
+        boolean increasement = (boolean) data;
```

Comment:
This code should also only be used if event is either EventConstant.UPDATE_APPT_NOTI_NUMBER or EventConstant.UPDATE_REQUEST_NOTI_NUMBER only.